openstates/tx/people.py

```diff
@@ -75,38 +75,53 @@ def _get_chamber_parties(self, chamber, term):
 
```

Comment:
PyCharm called it out, but in actual practice, it's not a problem. This is in a loop walking the cells of a table, and for each row it will always encounter the 3rd cell before the 7th.